nipype/external/tests/test_version.py

```diff
@@ -0,0 +1,33 @@
+import warnings
+
+import pytest
+
+from nipype.external.version import LooseVersion as Vendored
+
+with warnings.catch_warnings():
+    warnings.simplefilter("ignore")
+    try:
+        from distutils.version import LooseVersion as Original
+    except ImportError:
+        pytest.skip()
+
+
+@pytest.mark.parametrize("v1, v2", [("0.0.0", "0.0.0"), ("0.0.0", "1.0.0")])
+def test_LooseVersion_compat(v1, v2):
+    vend1, vend2 = Vendored(v1), Vendored(v2)
+    orig1, orig2 = Original(v1), Original(v2)
```

Comment:
should we catch this `DeprecationWarning` warning that occurs when initializing a distutils Version?

Reply:
Do you get one? I don't in 3.10.

Reply:
yeah, and i'm on 3.10.4...

```
DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  v = LooseVersion("0.0")
```

Reply:
```
$ pytest nipype/external/tests/test_version.py 
==================================================================== test session starts =====================================================================
platform darwin -- Python 3.10.4, pytest-7.1.1, pluggy-1.0.0
rootdir: /Users/mathiasg/code/nipype/nipype, configfile: pytest.ini
collected 2 items                                                                                                                                            

nipype/external/tests/test_version.py ..                                                                                                               [100%]

====================================================================== warnings summary ======================================================================
nipype/interfaces/io.py:1406
  /Users/mathiasg/code/nipype/nipype/interfaces/io.py:1406: DeprecationWarning: invalid escape sequence '\w'
    field_name = re.match("\w+", field_name).group()

nipype/interfaces/utility/base.py:232
  /Users/mathiasg/code/nipype/nipype/interfaces/utility/base.py:232: DeprecationWarning: invalid escape sequence '\w'
    """Change the name of a file based on a mapped format string.

../../.pyenv/versions/3.10.4/envs/nipreps/lib/python3.10/site-packages/_pytest/config/__init__.py:1252
  /Users/mathiasg/.pyenv/versions/3.10.4/envs/nipreps/lib/python3.10/site-packages/_pytest/config/__init__.py:1252: PytestConfigWarning: Unknown config option: env
  
    self._warn_or_fail_if_strict(f"Unknown config option: {key}\n")

external/tests/test_version.py::test_LooseVersion_compat[0.0.0-0.0.0]
external/tests/test_version.py::test_LooseVersion_compat[0.0.0-0.0.0]
external/tests/test_version.py::test_LooseVersion_compat[0.0.0-1.0.0]
external/tests/test_version.py::test_LooseVersion_compat[0.0.0-1.0.0]
  /Users/mathiasg/code/nipype/nipype/external/tests/test_version.py:18: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    orig1, orig2 = Original(v1), Original(v2)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
```

in any case, it's not a big deal

Reply:
Okay. I'll wrap it. Thanks for confirming.